kratos/processes/find_nodal_h_process.cpp

```diff
@@ -34,11 +35,14 @@ void FindNodalHProcess<THistorical>::Execute()
     }
 
     // Initialize NODAL_H values
-    #pragma omp parallel for
-    for(int i = 0; i < static_cast<int>(mrModelPart.Nodes().size()); ++i) {
-        auto it_node = mrModelPart.NodesBegin() + i;
-        SetInitialValue(it_node);
-    }
+
+    //Using Parallel Utilities
+    IndexPartition<std::size_t>(mrModelPart.Nodes().size()).for_each([&](std::size_t index){
+        SetInitialValue(mrModelPart.NodesBegin() + index);
+    });
+
+    //Or Using Variable_utils
```

Comment:
@philbucher can we use Variable utils here??? Compilation was successful, but whether will it work always ??
SetInitialValue() here is similar to SetVariable from Variable Utils

Reply:
no we can not, `SetInitialValue()` behaves differently depending on the class template

Reply:
Got it .